app/conode/start-conode.sh

```diff
@@ -9,22 +9,28 @@ main(){
   setup)
     if [ -f key.pub ]; then
       echo -e "\n*** Key.pub already exists - if you want to re-create, please delete it first\n"
+      exit
     else
-      ./conode keygen $2
-      cat key.pub | mail linus.gasser@epfl.ch
+      if ./conode keygen $2; then
+        echo Sending public-key to linus.gasser@epfl.ch
+        cat key.pub | mail linus.gasser@epfl.ch
```

Comment:
Are you sure about using mail ? It's not even configured on mine. 
The manual way is still the easiest way to do this. (Send email by hand).

Later, I think that when verification is done, we could make these operations automatic. Imagine the following workflow:
 - leader -> node node saying "Ok you're good to go,verification is OK, send me your public key now"
 - node -> leader : "this is my public key <adress> <publickey>"
 - leader computes the config.toml, waits for any other nodes if he wants then
    - leader -> node :" this is the config.toml you should use, run now please !"

But this is something related to bryan wishes of the tree being stateless also so it is only an idea to be incorporated into futur changes..